src/features/locators/Locator.tsx

```diff
@@ -95,14 +105,22 @@ export const Locator: React.FC<Props> = ({ element, currentPage, searchState, de
     }
   };
 
-  const handleLocatorClick: React.MouseEventHandler<HTMLDivElement> = (event) => {
-    const keyForMultiSelect = isMacPlatform(window) ? event.metaKey : event.ctrlKey;
+  const handleLocatorClick: React.MouseEventHandler<HTMLDivElement> = (evt) => {
+    const keyForMultiSelect = isMacPlatform(window) ? evt.metaKey : evt.ctrlKey;
```

Comment:
just curious, what makes you rename it?

Reply:
Just old habit to avoid using event word like a name, because of potential confusing with global variable.
But I've read that it'd been used in old browsers like IE, that isn't support anymore, so I can refuse from this habit now)